src/server/knexfile.ts

```diff
@@ -0,0 +1,44 @@
+// Update with your config settings.
+
+module.exports = {
```

Comment:
Creating a typescript file?? What's going on here. Please delete this file. I think you might have run some command in knex that initialized knex with a new config file. Make sure to read your PR through before creating it so you can catch files that might be created by accident.

Reply:
Deleted